Readme.md

```diff
@@ -143,16 +143,34 @@ $ LIBRARY_PATH="[gcc-path value]" LD_LIBRARY_PATH="[gcc-path value]" rustc +$(ca
 ## Env vars
 
 <dl>
-    <dt>CG_GCCJIT_INCR_CACHE_DISABLED</dt>
-    <dd>Don't cache object files in the incremental cache. Useful during development of cg_gccjit
-    to make it possible to use incremental mode for all analyses performed by rustc without caching
-    object files when their content should have been changed by a change to cg_gccjit.</dd>
+    <dt>CG_GCCJIT_DUMP_ALL_MODULES</dt>
```

Comment:
@antoyo: We should really remove this HTML and just write plain markdown...

Reply:
Please suggest the formatting to be used.